app/controllers/catalog_admin/csv_imports_controller.rb

```diff
@@ -38,6 +38,7 @@ def build_csv_import(params=nil)
     @csv_import = CSVImport.new(params) do |import|
       import.creator = current_user
       import.item_type = item_type
+      import.file_encoding = "detect"
```

Comment:
Why ?

Reply:
sinon les tests plantent

Reply:
![screenshot 2024-05-14 at 11 16 58](https://github.com/catima/catima/assets/417818/8a5440b1-e6e0-47ab-be38-01eb3608aeb5)

en haut sans le `import.file_encoding = "detect"`, en bas avec.

Reply:
Strange... I don't see what bootstrap or jquery have to do with it.